Changes

```diff
@@ -83,9 +83,13 @@ Compilers:
   review by Xavier Leroy)
 - PR#7026, GPR#288: remove write barrier for polymorphic variants without arguments
   (Simon Cruanes)
+- PR#7031: new warning, ambiguous guarded or-patterns (Luc Maranget,
+  Gabriel Scherer, report by Martin Clochard and Claude Marché).
 - PR#7067: Performance regression in the native compiler for long
   nested structures (Alain Frisch, report by Daniel Bünzli, review
   by Jacques Garrigue)
+- PR#6400: better error message for '_' used as an expression
```

Comment:
> Changes is better changed at the very end...

But it's not your change to Changes which is at the end, it's a correction to your change to Changes (which was itself done a commit before).
You should squash :)

http://existentialcomics.com/comic/9